services/movie_session.py

```diff
@@ -0,0 +1,59 @@
+from django.db.models import QuerySet
+from django.forms import DateTimeField
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: DateTimeField,
```

Comment:
The parameter `movie_show_time` should be of type `datetime`, not `DateTimeField`, as it represents a value, not a form field.